scripts/run_lint.sh

```diff
@@ -28,6 +28,9 @@ pylint --reports=n --output-format=colorized --rcfile=.pylintrc -j 0 ./app ./tes
 # http://stackoverflow.com/questions/6626351/how-to-extract-bits-from-return-code-number-in-bash
 display_result $? 2 "Pylint linting check"
 
+black -S --check app
+display_result $? 1 "Python code formatting check"
+
```

Comment:
I wonder if it's worth adding a pre-commit hook for this?

I do that on some other projects and it works quite well. I know some people really don't like it though.

Just needs a `pre-commit-config.yaml` with:
```
repos:
-   repo: https://github.com/ambv/black
    rev: stable
    hooks:
    - id: black
      language_version: python3
```